samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -44,6 +54,37 @@ static void sensor_cli_data_cb(struct bt_mesh_sensor_cli *cli,
 		}
 	} else if (sensor->id == bt_mesh_sensor_present_amb_light_level.id) {
 		printk("Ambient light level: %s\n", bt_mesh_sensor_ch_str(value));
+	} else if (sensor->id == bt_mesh_sensor_motion_sensed.id) {
+		int64_t motion_sensed = 0;
+
+		(void)bt_mesh_sensor_value_to_micro(value, &motion_sensed);
+		if (!!motion_sensed) {
+			is_occupied = true;
+			printk("Motion detected. Area is occupied.\n");
+			k_work_cancel_delayable(&motion_timeout_work);
```

Comment:
This looks a bit over-complicated. Why can't you handle this just like we already handle `presence_detected`, without this extra message 1 minute later?

Reply:
The motion sensor has the added feature according to the NLC profile that when motion is sensed, the area is to be considered occupied for a certain time which should be handled by the client. This is why the handling is a bit different than for presence detection. Is there still a part of the logic that you think is unnecessary? Or could you be more specific?

> When motion is detected, the space monitored by the sensor is considered occupied for a certain time. This time is not defined by the sensor, but instead by a model receiving the sensor messages.